src/main/java/net/glowstone/block/state/GlowDispenser.java

```diff
@@ -32,11 +33,34 @@ public static DispenseBehaviorRegistry getDispenseBehaviorRegistry() {
     public static void register() {
         // register all dispense behaviors
         DefaultDispenseBehavior bucketDispenseBehavior = new BucketDispenseBehavior();
+        ArmorDispenseBehavior armorDispenseBehavior = new ArmorDispenseBehavior();
```

Comment:
Move this line to right before you register the behavior for all of the new materials. (line 42 in current file)